smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/KafkaConsumer.java

```diff
@@ -181,4 +182,16 @@
     @CheckReturnValue
     Uni<Void> seekToEnd(Collection<TopicPartition> partitions);
 
+    /**
+     * @return the Uni emitting the group metadata of this consumer
+     */
+    Uni<ConsumerGroupMetadata> consumerGroupMetadata();
+
+    /**
+     * For each assigned topic partition reset to last committed offset or the beginning
+     *
+     * @return the Uni emitting {@code null} when the reset has been executed.
+     */
+    Uni<Void> resetToLastCommittedPositions();
+
```

Comment:
The existing methods are all annotated `@CheckReturnValue`, maybe these should too?